src/MessagingInput/MessagingInput.less

```diff
@@ -40,7 +40,7 @@
 
 button.MessagingInput--SendButton {
   // Sizing for this button is pretty particular
-  padding: 0.625rem @size_l 0.625rem;
+  padding: 0.6875rem 1.5rem 0.6875rem;
```

Comment:
`line-height: normal` is what we're missing here that would prevent different heights in different environments (as we've seen).